composer.json

```diff
@@ -3,12 +3,17 @@
     "license": "OPL-1.0",
     "require": {
         "php": "^8.1",
+        "jangregor/phpstan-prophecy": "^1.0",
```

Comment:
Why this one? We don't use prophecy, do we?

Reply:
true, removed it until we need it